keras_rcnn/models.py

```diff
@@ -30,15 +30,15 @@ def __init__(self, inputs, features, heads, rois):
 
         rpn_prediction = keras.layers.concatenate([rpn_classification, rpn_regression])
 
-        proposals = keras_rcnn.layers.object_detection.ObjectProposal(rois)([im_info, rpn_regression, rpn_classification])
-        anchors = keras_rcnn.layers.object_detection.ProposalTarget()([rpn_classification, gt_boxes, im_info])
+        proposals                        = keras_rcnn.layers.ObjectProposal(rois)([im_info, rpn_regression, rpn_classification])
```

Comment:
Maybe it's not necessary, but it'd be clearer to have `maximum_proposals=rois` instead of just `rois`
Also, ObjectProposal and ProposalTarget have im_info in different orders...we should be consistent. Could you switch it so that im_info comes at the end?